src/jit/morph.cpp

```diff
@@ -7629,7 +7629,7 @@ void Compiler::fgMorphTailCallViaHelper(GenTreeCall* call, void* pfnCopyArgs)
 
     if (call->gtCallThisArg != nullptr)
     {
-        GenTree* thisPtr    = nullptr;
+
```

Comment:
Useless empty line

Reply:
Thanks, I have dropped that commit, there was another use that I have missed there.